.eslintrc.json

```diff
@@ -3,7 +3,9 @@
     "plugin:@wordpress/eslint-plugin/recommended-with-formatting",
     "plugin:jest/recommended",
     "plugin:sitekit/main",
-    "plugin:react-hooks/recommended"
+    "plugin:react-hooks/recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings"
```

Comment:
As commented on ticket https://github.com/google/site-kit-wp/issues/3502#issuecomment-883490130
`plugin:import/recommended` pulls in `comma-dangle` as an error.  Odd

Reply:
Very odd. I'd like to see if we can figure out why that is happening to stick with `recommended` but we don't need to spend too much time on it.

Reply:
As mentioned [here](https://github.com/google/site-kit-wp/issues/3502#issuecomment-884168941) that's not the case, so let's use the recommended plugin instead.